### DISCUSSION TITLE
How can I self hosting hoppscotch in my local network

### DISCUSSION THREAD

**Discussion by hlxkurapika** [hoppscotch/hoppscotch#3018] [Q&A] [answered]
Hello, I have been watching hoppscotch for a long time, and very glad to found that hoppscotch support self-hosting. My company doesn't allow us to connect to internet for security, so I want to self-hosting hoppscotch in our local network to let my team can use hoppscotch **with workspace**, however, I found that the doc commented that we **have to** config a **third party OAuth** provider, but we could not connet to Github/Google/Microsoft or Firebase in our develop environment, Is there any solution to help us to solve this? I can write a custom OAuth service if necessary, hope your guidance. 
And forgive my poor english.
thank you.

**Comment by liyasthomas** [comment, marked as answer]
Follow instruction on documentation: https://docs.hoppscotch.io
